package.json

```diff
@@ -12,30 +11,33 @@
     "prepare:pods:data": "cpy initial-pod-data/* pods/example/ --flat",
     "start:pods": "npx @solid/community-server -c @css:config/file-no-setup.json -f pods",
     "reset:pods": "npx rimraf pods && npm run prepare:pods",
-    "build": "npx webpack",
-    "start": "npx webpack serve -c ./webpack.dev.config.js"
+    "build": "webpack .",
```

Comment:
Why is the use of `npx` removed?

Reply:
The dependencies were already installed in the dev dependencies, so both work here, It can also be placed back without any differences in functionality

Reply:
Didn't know that! 😄